src/const.h

```diff
@@ -461,15 +461,35 @@ enum item_t : uint16_t {
 	ITEM_WILDGROWTH_SAFE = 11099,
 
 	ITEM_BAG = 1987,
+	ITEM_SHOPPING_BAG = 23782,
 
 	ITEM_GOLD_COIN = 2148,
 	ITEM_PLATINUM_COIN = 2152,
 	ITEM_CRYSTAL_COIN = 2160,
+	ITEM_FORGGOTEN_COIN = 24774, // in-game store currency
```

Comment:
I dont think thats something we gonna allow in main repository.

Reply:
it is the official store currency, Tibia coins, but since we avoid using Tibia for names because of copyright issues, I named it Forgotten coin, it is already in items.xml as it

Reply:
Forggoten is wrong

Reply:
@EPuncker what he meant was that you had misspelled "forgotten", you wrote "forggoten".

Reply:
Just rename all instances to ``ITEM_STORE_COIN`` as it makes more sense. "Forgotten" is the emulators name.